packages/stores/src/price/pool.ts

```diff
@@ -48,80 +49,70 @@ export class PoolFallbackPriceStore
     return result;
   }
 
-  getPrice(coinId: string, vsCurrency?: string): number | undefined {
-    if (!vsCurrency) {
-      vsCurrency = this.defaultVsCurrency;
-    }
-
-    try {
-      const routes = this.intermediateRoutesMap;
-      const route = routes.get(coinId);
-      if (route) {
-        const pool = this.queryPool.getPool(route.poolId);
-        if (!pool) {
-          return;
-        }
+  readonly getPrice = computedFn(
+    (coinId: string, vsCurrency?: string): number | undefined => {
+      if (!vsCurrency) {
+        vsCurrency = this.defaultVsCurrency;
+      }
```

Comment:
So computedFn is just memoizing this, were we calling this function repeatedly a lot before?

Am in support of making it memoized / computed!

Reply:
Yeah basically everything is memoized, for some reason this wasn't. So if given the same input, it will return a cached input, which is an absolute need with how often components get rendered. However, with _all_ asset prices being computed from pool weights, the one-time (due to memo) recursion still causes quite a delay.

Reply:
A global ⌘⇧F on `computedFn` will come up with a _lot_ of results